samples/winapi_filefind/main.c

```diff
@@ -2,7 +2,7 @@
 #include <string.h>
 #include <fileapi.h>
 #include <winerror.h>
-#include <xboxrt/debug.h>
+#include <debug.h>
```

Comment:
Needs to be `hal/debug.h` now (as it was moved from `xboxrt/libc_extensions/debug.h` to `hal/debug.h`)?

Also needs review of other PRs which had received recommendation to just use `debug.h`.

Reply:
It works like this already, but I still changed the path to be explicit. I don't think it's worth the trouble to go through PRs (and SDL) just to change it to the full path.